bin/build-uc-server-docker

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -ex;
+
+current_dir=$(pwd)
+script_dir=$(dirname $0)
+docker_dir=$(dirname $(realpath $script_dir))
+
+echo "Changing directory to $docker_dir"
+cd $docker_dir
+
+if [[ -r Dockerfile ]]; then
+    docker buildx build --progress=plain --no-cache -t unitycatalog:0.1.0 .
```

Comment:
saw the version hardcoded in multiple places, let's try to use the version from version.sbt

Reply:
This is indeed something that needs to be changed.
I'll fix it asap.

Reply:
@vikrantpuppala  what's the usual approach for reading the version from sbt?
is it by reading the `version.sbt` file directly and parsing its content or is there an sbt function that can return the version?
In python i would use `hatch version` is there something similar for sbt ?

Reply:
There is a proposed method [here](https://alvinalexander.com/source-code/how-print-sbt-project-version-number-command-line/), but I want to verify if that's the expected way.
`sbt -Dsbt.supershell=false -error "print version"`

Reply:
We've done this by reading the file directory previously: https://github.com/unitycatalog/unitycatalog/blob/main/tests/test_tarball_generation.py#L29
but the command you shared should also be fine

Reply:
I've changed the versions but still need to test. Hopefully by tomorrow this conversation can be resolved.